### PR TITLE
instance_norm

### LINKED ISSUE #26408: instance_norm
- [ ] #22053

### PR DESCRIPTION
**Pull request by shehroz65**
# PR Description

## Related Issue

Close https://github.com/unifyai/ivy/issues/26409

## Checklist

- [x] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### Socials

Close #26408

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/norm.py [hunk 1]
```diff
@@ -4,6 +4,30 @@
...
+def batch_norm(
+    x,
+    weight=None,
+    bias=None,
+    running_mean=None,
+    running_var=None,
+    training=False,
+    momentum=0.1,
+    epsilon=1e-05,
+    name=None,
```
Comment (Sam-Armstrong) [inline ivy/functional/frontends/paddle/nn/functional/norm.py:18]:
```suggestion
    x,
    running_mean,
    running_var,
    weight,
    bias,
    training=False,
    momentum=0.9,
    epsilon=1e-05,
    data_format="NCHW",
    use_global_stats=None,
    name=None,
```
This should be the function signature, following the paddle docs here https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/batch_norm_en.html#batch-norm

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py [hunk 2]
```diff
@@ -10,6 +10,72 @@
 )
 
 
+# batch_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.batch_norm",
+    values_tuple=...,  # Generate appropriate test data for batch_norm
+    # Other hypothesis strategies, similar to the previous tests
+)
```
Comment (Sam-Armstrong) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py:18]: Please can you complete this test so it passes?

**Review by Sam-Armstrong — CHANGES_REQUESTED** [review]
Hi @shehroz65, I've left a couple of comments. I suggest we just stick to implementing batch_norm here - you can remove the instance_norm function and test here and create a new PR for it if you'd like.

**Comment by Ishticode** [comment]
Thank you verymuch for the PR. closing this due to lack of activity with changes requested remaining unresolved. Please feel free to reopen if you would like to continue working on this.  :)

**Pull request closed without merging**